tests/trinity/integration/test_trinity_cli.py

```diff
@@ -35,8 +35,8 @@ def async_process_runner():
 @pytest.mark.parametrize(
     'command',
     (
-        ('trinity',),
-        ('trinity', '--ropsten',),
+        ('trinity', '--trinity-root-dir .',),
+        ('trinity', '--trinity-root-dir .', '--ropsten',),
```

Comment:
Pretty sure this should be: `('trinity', '--trinity-root-dir', '.', '--ropsten',),`

Also: I think we should test trinity with & without the root dir specified:
```py
 ('trinity',),
 ('trinity', '--trinity-root-dir .',),
```